src/frontend/pages/blogs/BlogPage.js

```diff
@@ -0,0 +1,206 @@
+  width: 95px;
+  height: 36px;
+  border: 1px solid ${(props) => props.theme.colours.yellows.yellow2};
+  font: ${(props) => props.theme.fonts.bold18};
+  line-height: 23px;
+  background-color: ${(props) => props.theme.colours.yellows.yellow2};
+  color: ${(props) => props.theme.colours.blacks.black3};
+  box-sizing: border-box;
+  border-radius: 15px;
+  cursor: pointer;
+
+  &:hover {
+    background-color: ${(props) => props.theme.colours.greys.grey2};
+    border: 1px solid ${(props) => props.theme.colours.greys.grey2};
+  }
+`;
+
+const TableLabelHeader = styled.span`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 30px;
+  margin-bottom: 0;
+
+  @media only screen and (max-width: ${({ theme }) => theme.breakpoints.md}px) {
+    justify-content: center;
+    flex-direction: column;
+    margin-bottom: 20px;
+  }
+`;
+
+const LinkButton = styled.button`
+  color: black;
+  text-decoration: none;
+`;
+
+const TableHeader = styled.p`
+  font: ${({ theme }) => theme.fonts.bold36};
+  margin: 10px;
+`;
+
+const PopUpOverlay = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color:rgb(64,64,64);
+  opacity:0.6;
+`;
+
+const PopUp = styled.div`
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  transform: translate(-50%, -50%);
+  border: 5px solid #FFFFFF;
+  padding: 10px;
+  width: 60vw;
+  height: 35vh;
+  background-color: white;
+  opacity: 1 !important;
+  z-index: 10;
+  border-radius: 10px;
+  box-shadow: 2px 10px 13px #888888;
+`;
+
+const CancelButton = styled.button`
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+  color: black;
+  background-color: white;
+  border: none;
+  text-decoration: none;
+`;
+
+const Close = styled.img`
+  width: 30px;
+  height: 30px;
+  cursor: pointer;
+`;
+
+const SummaryTitle = styled.h5`
+  font: ${({ theme }) => theme.fonts.bold36};
+  margin: 10px;
+`;
+
+const SummaryDesc = styled.p`
+  padding: 0 40px 20px 40px;
+  font: ${({ theme }) => theme.fonts.light18};
+`;
+
+const headers = [
+  {id: 'title', value: 'Blog'},
+  {id: 'updatedAt', value: 'Date'},
+  {id: 'author', value: 'Author'},
+  {id: 'link', value: 'Medium Link'}
+];
+
+const RowComponent = ({ title, updatedAt, author, link }) => (
+  <>
+    <TableCell>
+      <TextBold>{title}</TextBold>
+    </TableCell>
+    <TableCell>{updatedAt}</TableCell>
+    <TableCell>{author}</TableCell>
+    <TableCell>
+      <LinkButton as="a" href={link}>Link</LinkButton>
+    </TableCell>
+  </>
+);
+
+const BlogPage = () => {
+  const { blogInfo } = useBlogInfo();
+  const history = useHistory();
+  const [popup, setPopup] = useState(<></>);
+
+  const blogs = blogInfo?.posts?.map((blog) => ({
+    id: blog.id,
+    title: blog.title,
+    author: blog.author,
+    updatedAt: blog.date,
+    link: blog.link,
+    summary: blog.summary
+  }));
```

Comment:
Out of curiousity, what happens when we are in the middle of fetching data and waiting for the data to arrive ? Are we just showing blank table to the user ?

You don't have to handle this for this Pull request, but this may be helpful for such situations:
https://www.youtube.com/watch?v=Vspeudp-M9k

If you think you want to try this out and build a loading screen this term, by all means you can create a separate ticket for it :)